subprojects/platform-play/src/testFixtures/resources/org/gradle/play/integtest/fixtures/app/basicplayjavaapp/app/views/main.scala.html

```diff
@@ -0,0 +1,15 @@
+@(title: String)(content: Html)
+<!DOCTYPE html>
+<html>
+    <head>
+        <title>@title</title>
+        <link rel="stylesheet" media="screen" href="@routes.Assets.at("stylesheets/main.css")">
+        <link rel="shortcut icon" type="image/png" href="@routes.Assets.at("images/favicon.png")">
+        <script src="@routes.Assets.at("javascripts/hello.js")" type="text/javascript"></script>
+    </head>
+    <body>
+@content
+      <hr/>
+@Calendar.getInstance().toString()
```

Comment:
Use a class from `java.util.*` which isn't available when using the Scala imports.